Checkpoint3/Program.cs

```diff
@@ -0,0 +1,236 @@
+                "quit:         exit program\n");
+
+                //asks user what they want to do
+                Console.WriteLine("What would you like to do?");
+                //stores input
+                input = Console.ReadLine().ToLower();
+                //switch statement that decides what to do based on user input
+                switch(input)
+                {
+                    //case for adding an item to database
+                    case "add":
+                        AddItem();
+                        break;
+                    //case for deleting an item from database
+                    case "delete":
+                        DeleteItem();
+                        break;
+                    //case for marking an item done in database
+                    case "mark done":
+                        MarkItem();
+                        break;
+                    //case to list pending items
+                    case "list pending":
+                        ListItems(false,true);
+                        break;
+                    //case to list done items
+                    case "list done":
+                        ListItems(true,true);
+                        break;
+                    //case to list all items
+                    case "list all":
+                        ListItems(true,false);
+                        break;
+                    //if user enters anything other than an available case, lets user know
+                    default:
+                        Console.WriteLine("incorrect input entered");
+                        break;
+                }
+                //clears console before while loop resets
+                Console.Clear();
+            }
+            
+        }
+
+        static void AddItem()
+        {
+            Console.WriteLine("Enter a description for your task:");
+            string itemDescription = Console.ReadLine();
+            theDao.add(itemDescription);
+            itemList = theDao.list();
+        }
+        static void DeleteItem()
+        {
+            foreach(Item eachItem in itemList)
+            {
+                Console.WriteLine(eachItem);
+            }
+            Console.WriteLine("\nEnter item to delete by Id:");
+            int itemToDelete = Convert.ToInt16(Console.ReadLine());
+            theDao.remove(itemToDelete);
+            itemList = theDao.list();
+        }
+        static void MarkItem()
+        {
+            foreach(Item eachItem in itemList)
+            {
+                Console.WriteLine(eachItem);
+            }
+            Console.WriteLine("Enter item Id to mark done:");
+            int itemToMark = Convert.ToInt16(Console.ReadLine());
+            theDao.markDone(itemToMark);
+            itemList = theDao.list();
+        }
+        static void ListItems(bool statusChecked, bool checkingSpecific)
+        {
+            foreach(Item listedItem in itemList)
```

Comment:
instead of using the static 'itemList' here, i would have gotten a "fresh" list from the dao (`theDao.list()`)